src/lib/bmp/bmp.h

```diff
@@ -0,0 +1,53 @@
+/**
+  * @file bmp.h
+  * @brief lib for read and draw BMP
+  * @author Filipp Chubukov
+  * @version
+  * @date 03.07.2019
+*/
+
+#ifndef LIB_BMP_H_
+#define LIB_BMP_H_
+
+/**
+  * @brief check BMP_head
+  *
+  * @param fd File descriptor
+  *
+  * @return -1 if not BMP and 0 if BMP
+*/
+extern int check_bmp(int fd);
+
+/**
+  * @brief fill struct bmp_header
+  *
+  * @param fd File descriptor
+  * @param bmp_header - buffer for bmp_header
+  *
+  * @return -1 if error and 0 if successed
+*/
+extern int get_head_bmp(int fd, bmp_header *bmp_head);
+
+/**
+  * @brief fill buffer with image
+  *
+  * @param fd File descriptor
+  * @param bmp_head - buffer for bmp header
+  * @param image - buffer for image, big enough for image
+  *
+  * @information colors in image are reversed to BGR
+  *
+  * @return -1 if error and 0 if successed
+*/
+extern int get_image_bmp(int fd, bmp_header *bmp_head, uint8_t *image);
+
+/**
+  * @brief draw BMP in framebuffer
+  *
+  * @param fd File descriptor
+  *
+  * @return Negative error code or zero if succeeded
+*/
+extern int draw_fb_bmp(int fd);
```

Comment:
I think it's passing file name instead of fd is a better solution

Reply:
just in draw functions or in all from header

Reply:
done in bmp

Reply:
done in png